### ISSUE TITLE
Missing <h1> on events page

### ISSUE THREAD

**Issue by tsidel** [ThatConference/that.us#1084]
**Describe the bug**
The events page is missing a first level heading <h1>

**To Reproduce**
1. Go to https://that.us/events/
2. View source
3. No <h1 is found.

**Expected behavior**
"Events at THAT" should be wrapped in an <h1> tag. Screen reader users can navigate a page according to its headings, listen to a list of the headings, and skip to selected headings to begin reading at that point. H1s can also be used (along with the <title> tag for Search Engines). (In this case, Google is reading from the <Title> tag)

**Additional context**
See these links for more details:

- https://www.w3.org/WAI/tutorials/page-structure/headings/

- https://webaim.org/techniques/semanticstructure/#contentstructure

**Also, please do not work on this yet, saving this issue for the That ONLINE November 15 edition**

I'm also looking to see if there are other pages are missing h1 tags.
